src/structured-data/structured-data-tags.js

```diff
@@ -352,6 +363,10 @@ export function StructuredDataTags({structuredData = {}}, config, pageType, resp
     if(structuredData.enableNewsArticle !== 'withoutArticleSchema') {
       return ldJson("Article", articleData);
     }
+
+    if(pageType === 'story-page' && story['story-template'] !== 'video') {
+      return ldJson("ImageObject", generateArticleImageData(structuredData, image, story, publisherConfig))
+    }
     return {}
```

Comment:
Can we have a parent ticket for these changes?

Reply:
Any update on this? Because in this ticket, we're adding the `ImageObject` for story page and `isPartOf` also. Can we create a ticket for this? Just go through with @theanandcom also whether adding an `ImageObject` schema to story is a good SEO thing.

Reply:
Removed from this branch